tasmota/my_user_config.h

```diff
@@ -395,7 +395,9 @@
                                                  // Enabled by default on ESP32 and variants
 
 // -- ESP-NOW -------------------------------------
-//#define USE_TASMESH                              // Enable Tasmota Mesh using ESP-NOW (+11k code)
+#define USE_TASMESH                              // Enable Tasmota Mesh using ESP-NOW (+11k code)
```

Comment:
Please leave all the below lines commented in your PR
Users wanting to enable the feature will add them in their user_config_override.h